package.json

```diff
@@ -41,11 +42,12 @@
     "lint-staged": "^10.5.4",
     "mini-css-extract-plugin": "^0.9.0",
     "npm-run-all": "^4.1.5",
+    "postcss": "^7.0.35",
     "postcss-loader": "^3.0.0",
     "prettier": "^2.2.1",
     "rimraf": "^3.0.2",
     "serve": "^11.3.2",
-    "tailwindcss": "^1.6.2",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.1.0",
```

Comment:
**question**: I don't understand this construction, why is this needed?

Reply:
https://tailwindcss.com/docs/installation#post-css-7-compatibility-build

Reply:
Thanks for providing the explanation, now I understand.